content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,54 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
+It also parses and displays live metrics and plots sent by [DVCLive]; these
+don't need to be pushed to the Git repositories.
```

Comment:
I think this is included in the "and metrics" part. We didn't even mention DVC earlier in this paragraph so why mention DVCLive? Too specific. The wording I suggested above also prevents the need to clarify about not having to push to Git. So I'd remove it from here
```suggestion
```
But maybe we can add this note somewhere else further down... Let me 👀

Reply:
We need to mention this here coz otherwise it's not clear that live metrics are not "found in your ML project repositories".